ulib/c_libax/src/resource.c

```diff
@@ -0,0 +1,60 @@
+#include <errno.h>
```

Comment:
Remove `errno` in `<errno.h>` to avoid the warning of unused variables.

Reply:
`ulib/c_libax/src/network.c` uses errno

Reply:
Define `errno` in `errno.c`, and use `extern unsigned int errno` in `errno.h`.